package/src/theme/defaultTheme.js

```diff
@@ -256,6 +256,53 @@ export const rawMuiTheme = {
         }
       }
     },
+    MuiChip: {
+      root: {
+        fontSize: defaultFontSize * 0.875,
+        letterSpacing: 0.28
```

Comment:
is there a letter-spacing value in the default theme that can be used here?

Reply:
@cassytaylor - Should we make a variable for "letter spacing for small size text" aka 0.28px.

Reply:
@willopez Added 2 variables: `captionLetterSpacing` and `smallFontIconSize`. I think the height of 30 is okay without being a variable cos it's unclear if we'd ever re-use that key/value.